department-of-veterans-affairs/va.gov-cms
# VAGOV-1568 migrate main nav

This got lost in the shuffle last sprint, but here it is. 

To test:

1. Go to /admin/config/structure/va_gov_migrate/truncate-menu to delete all the items currently in the main menu.
2. Run the 'Import menu links for health care benefits main menus' migration.

The main menu should now match the one on va.gov.
